src/cart/src/gurt/dlog.c

```diff
@@ -346,6 +346,25 @@ static void dlog_cleanout(void)
 #endif
 }
 
+static __thread int	 pre_err;
+static __thread int	 pre_err_line;
+static __thread uint64_t pre_err_time;
+
+#define dlog_print_err(err, fmt, ...)				\
+do {								\
+	struct timeval	_tv;					\
+								\
+	gettimeofday(&_tv, NULL);				\
+	if (pre_err_line == __LINE__ &&				\
+	    pre_err == (err) &&	_tv.tv_sec <= pre_err_time)	\
```

Comment:
The "_tv.tv_sec <= pre_err_time" can never be satisfied, right? It should compare with "pre_err_time + delta"?

Reply:
sorry I don't understand, it can be satisfied every second right?
see L361: pre_err_time = _tv.tv_sec;

Reply:
yes, I didn't notice it's in sec.